flashcard_samples.csv

```diff
@@ -0,0 +1,39 @@
+term, definition
```

Comment:
@utahrim @saraho-shea @RebeccaKleinberg 

I think we have to delete the spaces after commas. For example, in line 1 it should be:

`term,definition`

And line 2 should read:

`alias,To create a second name for the variable or method`

Otherwise, we will have spaces inserted in weird areas. (For example, our definition within the class would start with a space).